js/stores/field-store.ts

```diff
@@ -1,18 +1,20 @@
 import * as THREE from "three";
+import { Rock } from "../plates-model/crust";
 import FieldBase from "../plates-model/field-base";
 import { IFieldsOutput } from "../plates-model/model-output";
 
 export default class FieldStore extends FieldBase {
   // Never ever use @observable decorator here. There are too many fields being used and simple setting of a property
   // value will take too much time (it's been tested).
-  elevation: number | null = null;
-  normalizedAge: number | null = null;
+  elevation = 0;
+  normalizedAge = 0;
   boundary = false;
   earthquakeMagnitude = 0;
   earthquakeDepth = 0;
   volcanicEruption = false;
   force = new THREE.Vector3();
   originalHue: number | null = null;
+  rockType: Rock = 0;
```

Comment:
Again, just for documentation, using `Rock.Sediment` might be nice.

Reply:
Thanks, fixed in the last commit.